cmd/aries-agentd/readme_aries-agentd.txt

```diff
@@ -0,0 +1,40 @@
+Building Aries-agentd
```

Comment:
move it under docs similar to https://github.com/hyperledger/aries-framework-go/blob/master/docs/demo/openapi_demo_instructions.md

Reply:
Add link in root readme file similar to  https://github.com/hyperledger/aries-framework-go/blob/master/README.md#running-demo-locally

Reply:
Done!